os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -235,10 +235,12 @@ static void rtl8730e_control_backlight(uint8_t level)
 		/* TO-DO: Move LCD IC Power ON flow */
 		InterruptEn(lcdc_irq_info.num, lcdc_irq_info.priority);
 	}
-#if defined(CONFIG_LCD_ST7701SN)
+#if defined(CONFIG_LCD_PWM_INVERSION)
```

Comment:
It is obvious that the backlight turns off when the PWM value is 0.
Even if user don't set a pwm mode additionally, it seems that it operates in the normal mode by default.
So I think you can remove a configuration for the basic operation and handle it when PWM_INVERSION is enabled only for better management.

#if defined(CONFIG_LCD_PWM_INVERSION)
...
#else
...
#endif